tests/unit/lms/resources/_lti_launch_test.py

```diff
@@ -16,7 +16,7 @@ def test_it_allows_LTI_users_to_launch_LTI_assignments(
         pyramid_config.testing_securitypolicy("TEST_USERNAME", groupids=["lti_user"])
         pyramid_config.set_authorization_policy(policy)
 
-        context = resources.LTILaunchResource(pyramid_request)
+        context = LTILaunchResource(pyramid_request)
 
```

Comment:
I can't see what this is doing for us other than making the tests a bit more long winded